variables.tf

```
@@ -15,6 +15,12 @@ variable "cloudwatch_logs" {
   description = "Whether or not to configure a CloudWatch log group"
 }
 
+variable "cloudwatch_logs_retention" {
+  type        = number
+  default     = 14
+  description = "Specifies the number of days you want to retain log events in the specified log group. Possible values are: 1, 3, 5, 7, 14, 30, 60, 90, 120, 150, 180, 365, 400, 545, 731, 1827, and 3653."
```

I prefer to keep these comments a bit more concise: 

```suggestion
  description = "Specifies the number of days you want to retain log events"
```